src/Microsoft.CodeQuality.Analyzers/Core/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -126,4 +126,1221 @@
   <data name="DoNotUseCountWhenAnyCanBeUsedTitle" xml:space="preserve">
     <value>Do not use Count() when Any() can be used</value>
   </data>
+  <data name="AvoidAsyncVoidTitle" xml:space="preserve">
```

Comment:
Probably not related to this PR, but all the async related rule prototypes/stubs have been removed from this repo and so the corresponding resource strings are redundant. We should eventually delete them.